tools/hugepages-machineconfig-generator/hugepages-machineconfig-generator.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"log"
+
+	"k8s.io/utils/pointer"
+
+	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/hugepages"
+	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components"
+)
+
+const nodeRole = "worker"
+
+func main() {
+	hpSize2M := performancev2.HugePageSize("2M")
```

Comment:
please change such as:

- this tool supports three command line flag: `-n` for the node role (vs hardcoded), `-o` for the output file (possibly stdout), `-i` for the input file (possibly stdin)
- this tool emits the relevant manifests as yaml files
- this tool reads a performance profile yaml, but ignores everything but the hugepages definition

so we can run something like
```
cat some-perf-profile.yaml | hugepages-machineconfig-generator -n worker-cnf -i - -o - | oc apply -f -
```

Reply:
even better: `stdin` should be the default for `-i` and `stdout` should be the default for `-o`

Reply:
modified, thanks!